### ISSUE TITLE
As a reader 👁️ I should know the total ➕ rows in a table 🔠 looking at the last row number 🔟

### ISSUE THREAD

**Issue by mattiapontonio** [mdn/mdn-minimalist#446]
TBD

**Comment by hamishwillee** [comment]
Minimalist team, in summary, Mattia is suggesting that it might be an accessibility improvement to provide the total number of table rows in a large table (for use by screen readers). 

I can see how this might be useful (though perhaps screen readers already do this). However this would not be something that an author should have to do. 

I closed the content-side issue because IMO it is not something an author should have to think about. However it might be amenable to automation.

**Comment by mattiapontonio** [comment]
> However this would not be something that an author should have to do.

That's true 👍

**Comment by schalkneethling** [comment]
Moving this to the Yari repo as this will need an automated solution should we do this in the future. With that in mind, Yari is the best place for this. Thanks, @mattiapontonio
